README.md

```diff
@@ -5,93 +5,109 @@
 
-- `/docs/` - Contiene los archivos de Markdown para los documentos. Personaliza el orden de la barra lateral de documentos en  `sidebars.js`. Se pueden encontrar más detalles en la [guía de documentos](https://v2.docusaurus.io/docs/markdown-features).
-- `/src/` - Archivos que no son de documentación como páginas o componentes personalizados de React. No tiene que colocar estrictamente sus archivos que no son de documentación aquí, pero colocarlos en un directorio centralizado facilita la especificación en caso de que necesite hacer algún tipo de procesamiento.
-- `/src/pages` - Cualquier archivo dentro de este directorio se convertirá en una página de sitio web. Se pueden encontrar más detalles en la [guía de páginas](https://v2.docusaurus.io/docs/creating-pages).
-- `/static/` - Directorio estático. Cualquier contenido dentro de aquí se copiará en la raíz del directorio de compilación final `build`.
-- `/docusaurus.config.js` - El archivo de configuración para el sitio. Este es el equivalente de siteConfig.js en Docusaurus 1.
-- `/package.json` - Un sitio web de Docusaurus es una aplicación de React. Puede instalar y usar cualquier paquete npm que desee en ellos.
-- `/sidebar.js` - Utilizado por la documentación para especificar el orden de los documentos en la navegación lateral.
+## Tech Stack
+- React.js - React 16✨, React Router 5
+- Material-ui/core - Componentes de React para un desarrollo web más rápido y fácil, biblioteca-autenticador-universal - Una biblioteca para permitir que las aplicaciones utilicen fácilmente diferentes proveedores de autenticación
+- Core - Componentes de React para un desarrollo web más rápido y sencillo
+- Lint- ESlint
+- Estilos - Tema Material-UI (personalizable)
+- Docusaurus - Herramienta de código abierto para construir, implementar y mantener la documentación web.
 
+### Empezando
 
-### Antes de empezar
-Es necesario contar con: 
-* git
-* node.js
+Dé las instrucciones necesarias para ejecutar el proyecto localmente, por ejemplo:
 
+#### **Requisitos Previos**
 
-### Instrucciones para ejecutar el portal de forma local en Windows 10
+- `yarn`
+- `git`
+- `node.js`
 
+#### **Instalación**
 
-1. Clone el repositorio
+1. Clona el repositorio
 
-```bash
-$ git clone https://github.com/eoscostarica/guias.eoscostarica.io.git
-```
+  `
+  git clone https://github.com/eoscostarica/guias.eoscostarica.io.git
+  `
 
-2. Ubíquese al directorio apropiado
+2. Mover el directorio apropiado
 
-```bash
-$ cd guias.eoscostarica.io
-```
+  `
+  cd guide.eoscostarica.io.git
+  `
 
-3. Instale las dependencias:
-```bash
-$ yarn install 
-```
+3. Instale los paquetes de yarn
 
-4. Compile el proyecto:
-```bash		
-$ yarn build 
-```
+  `
+  yarn
+  `
+
+4. Ejecute el proyecto
 
-5. Ejecute el servidor local:
-```bash		
-$ yarn serve 
+  `
+  yarn start
+  `
+
+### Estructura de archivos
+
+```sh
+guias.eoscostarica.io/
+├── contracts ..................... Contrato Inteligente de Activos Simples
```

Comment:
this folder is from a different project , we need to adjust this example to the current project

Reply:
```
root-directory
├── Dockerfile
├── README.md
├── docker-compose.yml
├── docs
│   ├── doc1.md
│   ├── doc2.md
│   ├── doc3.md
│   ├── exampledoc4.md
│   └── exampledoc5.md
└── website
    ├── blog
    │   ├── 2016-03-11-blog-post.md
    │   ├── 2017-04-10-blog-post-two.md
    │   ├── 2017-09-25-testing-rss.md
    │   ├── 2017-09-26-adding-rss.md
    │   └── 2017-10-24-new-version-1.0.0.md
    ├── core
    │   └── Footer.js
    ├── package.json
    ├── pages
    ├── sidebars.json
    ├── siteConfig.js
    └── static
```

Reply:
OF course!!